justfile

```
@@ -20,7 +20,6 @@ init:
 
 # Install all dependencies. Run it once.
 setup:
```

```suggestion
setup: init
```

This just runs `git submodule update --init --recursive` which is an extremely low cost check if it is already up to date. This just ensures that all dependencies are synced with the versions in git. Having this might also allow you to skip a step in your CI flow, as it essentially enforces that hash checked out == hash committed for a given submodule